### ISSUE TITLE
WCAG validation failure for aria described by after fixing errors on forms.

### ISSUE THREAD

**Issue by luisluix** [aspnet/jquery-validation-unobtrusive#75]
https://dotnetfiddle.net/HiDCu5

Step 1; click submit
Step 2: put something in textbox 1
Step 3: inspect elements

If you check the "aria-described by" of the elements, input1 aria-described by points to "inputtext-error", but since we already fixed the error, the element with the "inputtext-error" ID no longer exists. This is what is causing the WCAG error. When we do step 2, aria-described by on input1 should be removed as well.

**Comment by mkArtakMSFT** [comment]
@jbagga, can you please look into this? Thanks!

**Comment by jbagga** [comment]
Thanks @luisluix 
This seems to be a dupe of https://github.com/jquery-validation/jquery-validation/issues/2094 as you are aware of already :)
Closing for now.
